.github/workflows/build_and_test.yml

```diff
@@ -138,6 +138,20 @@ jobs:
       stepName: 'rust-check'
     secrets: inherit
 
+  rust-doc-check:
+    name: rust-dock-check check
+    needs: ['changes', 'build-next']
+    if: ${{ needs.changes.outputs.docs-only == 'false' }}
+
+    uses: ./.github/workflows/build_reusable.yml
+    with:
+      needsRust: 'yes'
+      skipInstallBuild: 'yes'
+      skipNativeBuild: 'yes'
+      afterBuild: ./scripts/deploy-turbopack-docs.sh
```

Comment:
Why call the bash script if we only need one line from it?

Reply:
It's mostly copy-paste https://github.com/vercel/next.js/blob/a3f4c35d972d7a1b3ca71aba09d7c934aae4789e/scripts/deploy-examples.sh#L4, also we invoke script for 2 places so can cosolidate rustdoc invocation in a script.